File: cpp/funkcje.cpp
/*
 * funkcje.cpp
 */


#include <iostream>

using namespace std;

void dodaj(int a, int b) // void -nie zwraca wyniku; całość - deklaracja funkji
{
    cout << "Suma: " << a + b << endl;
}
int odejmij (int l1, int l2)
{
    return l1 - l2; //zwraca wynik. RETURN NIE WYSTĘPUJE Z VOID!
}
void mnoz(int x, int y)
{
    cout << "Iloczyn: " << x * y << endl;
}
int dziel (int c, int d)
{
    if(d == 0)
    {
        cout << "Nie dziel przez 0!" << endl;
        return 0;
    }
    return c / d;
}

int main(int argc, char **argv)
{
    int a,b;
    a = b = 0;
    
    cout << "Podaj 2 liczby: " << endl;
    cin >> a >> b;
    
    dodaj(a, b);  //wywołanie funkcji, używanie
    cout << "Różnica: " << odejmij(a, b) << endl;
    mnoz(a,b);
    cout << "Iloraz: " << dziel (a, b) << endl;
    
    
    return 0;
}

File: Python/funkcje.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def sumuj(a, b):
    wynik = a + b
    return wynik


def odejmij(a, b):
    wynik = a - b
    return wynik


def iloczyn(a, b):
    wynik = a * b
    return wynik


def iloraz(a, b):
    wynik = a / b
    return wynik


def main(args):
    a = int(input("Podaj 1. liczbę"))
    print(a)
    b = int(input("Podaj 2. liczbę"))
    print(b)

    print("suma: ", sumuj(a, b))
    print("różnica: ", odejmij(a, b))
    print("iloczyn: ", iloczyn(a, b))
    print("iloraz: ", iloraz(a, b))

    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
